runs/admin.py

```diff
@@ -20,5 +20,5 @@ class ContentChannelRunAdmin(admin.ModelAdmin):
     def channel_id(self, obj):
         return obj.channel.channel_id
     inlines = [ChannelRunStageInline, ]
-    list_display = ('run_id', 'channel_id', 'chef_name', 'logfile')
+    list_display = ('run_id', 'channel_id', 'chef_name', 'created_at')
```

Comment:
This list populates the Django `/admin/` interface (used for debugging), so it's OK.